modules/virt-about-using-virtual-gpus.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-mediated-devices.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-using-virtual-gpus_{context}"]
+= About using virtual GPUs with {VirtProductName}
+
+If you have a host with a compatible graphics processing unit (GPU), you can use it to create virtual GPUs (vGPUs). {VirtProductName} can automatically create vGPUs and other _mediated_ devices if an administrator provides configuration details in the `HyperConverged` custom resource (CR). This automation is especially useful for large clusters.
+
+[NOTE]
+====
+Refer to your hardware vendor's documentation for functionality and support details.
+====
+
+Mediated device:: A physical device that is divided into one or more virtual devices. A vGPU is a type of mediated device (mdev); the performance of the physical GPU is divided among the virtual devices. You can assign mediated devices to one or more virtual machines (VMs), though some GPUs do not support multiple guests.
+
+GPU passthrough:: Allows a virtual machine to use a host's GPU as if it were physically attached to the guest.
+
+[id="configuration-overview_{context}"]
+== Configuration overview
+
+To configure mediated devices in the `HyperConverged` CR, an administrator must:
+
+* Expose the mediated devices to the cluster by listing their details in the `spec.permittedHostDevices` stanza.
+
+* List the types of mediated devices that you want {VirtProductName} to create in the `spec.mediatedDevicesConfiguration` stanza.
```

Comment:
We need to reverse the order here,
a) We should first create the "mediated devices" and then
b) Expose the "mediated devices" to the cluster